src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/cloud/deploy/CloudDeployFrontendCommand.kt

```diff
@@ -106,8 +106,19 @@ class CloudDeployFrontendCommand(
 
     override fun execute(output: ColorWriter): ExitCode {
         try {
+            val saClientId = System.getenv("NPL_SA_CLIENT_ID") ?: System.getProperty("NPL_SA_CLIENT_ID")
```

Comment:
Nit: I would make these `NPL_SERVICE_ACCOUNT_CLIENT_ID` in order to never confuse anyone ("what is 'SA'?", someone will inevitably ask)